cav_msgs/msg/RoadwayObstacle.msg

```diff
@@ -2,7 +2,7 @@
 # RoadwayObstacle.msg
 # 
 # Describes the location and geometry of an obstacle or vehicle in the roadway
-# Obstacles are defined relative to the route with downtrack and crosstrack distances, and lane numbers
+# Obstacles are defined relative to the lanelet with downtrack and crosstrack distances, and lanelet id.
 # The obstacle's size and velocity are described in a frame located on it's current route segment
 # The frame is a FRD frame located on the uptrack waypoint of a segment. 
 # With +x pointing along the segment and +y pointing to the right
```

Comment:
The above 3 lines are left over from CARMA2. As we no longer have a route segment to reference we need to update this comment. I think it is sufficient to delete the above 3 lines.